app/policies/aal3_policy.rb

```diff
@@ -0,0 +1,50 @@
+class AAL3Policy
```

Comment:
I think we got rid of this file right?

Reply:
We did, I botched a merge and brought it back. Will remove